### ISSUE TITLE
[SPIKE] Figure out the responsibility between express middleware and LB4 routing and sequencing

### ISSUE THREAD

**Issue by hacksparrow** [strongloop/loopback-next#1291]
## Description / Steps to reproduce / Feature proposal
Mount LB4 apps (REST servers) as Express (and Express-compatible) middleware

## Current Behavior
None

## Expected Behavior
https://github.com/strongloop/loopback-next/pull/1082#issuecomment-384580197

## Acceptance Criteria
LB4 apps should be mountable as Express or Express-compatible middleware.
- [ ] Figure out how to have everything before and after the REST routing handled by express http transport-wise
- blocked by #1292 and #1293

**timeboxed to 1 week**

_See [Reporting Issues](http://loopback.io/doc/en/contrib/Reporting-issues.html) for more tips on writing good issues_

**Comment by shimks** [comment]
Technically, the acceptance criteria is already done. @hacksparrow Would you consider the issue non-relevant or do you have specific items in mind that allow express-specific aspects to be compatible with lb4?

Suggested criteria:
- change `(req, res)` pattern to use `httpContext` (with a different name)
- and potentially more

**Comment by bajtos** [comment]
For DP3, we agreed with @raymondfeng that we are not going to expose Express routing/middleware to app developers yet. I am removing this story from DP3 release and epic.
